tests/benchmarks/utils.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from git import Repo
+
+CLONE_TO_DIR = f"{os.path.dirname(__file__)}/../../../"
```

Comment:
Shouldn't this just be the home file (Path.home())?

Reply:
This is cloning outside the Mentat repo, right? What if the repo is Mentat, like in your new benchmark? won't there be a conflict?

Maybe we should make a directory inside mentat, something like `repos_for_benchmarks`?

Reply:
`mentat/tests/benchmarks/repos`?